boilerplate/openshift/golang-osd-operator/standard.mk

```diff
@@ -137,6 +137,10 @@ olm-deploy-yaml-validate: python-venv
 prow-config:
 	${CONVENTION_DIR}/prow-config ${RELEASE_CLONE}
 
+.PHONY: codecov-secret-mapping
+codecov-secret-mapping:
```

Comment:
should this target depend on `prow-config`?

Reply:
Is the plan to create the secret mapping in a different openshift/release PR? How would someone create a single PROW config and secret mapping in a single PR?